Harvest.Web/ClientApp/src/Tickets/TicketListContainer.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from "react";
+import { Link, useHistory } from "react-router-dom";
+
+import { Ticket } from "../types";
+import { TicketTable } from "./TicketTable";
+
+interface Props {
+  projectId: any;
+}
+
+
+export const TicketListContainer = (props: Props) => {
```

Comment:
This isn't a container anymore now that it is being passed props and it's top level.  Better to just call it `TicketList` or similar to make room from your new container.